src/main/java/org/broadinstitute/hellbender/cmdline/StandardArgumentDefinitions.java

```diff
@@ -36,6 +36,7 @@ private StandardArgumentDefinitions(){}
     public static final String DISABLE_SEQUENCE_DICT_VALIDATION_NAME = "disable-sequence-dictionary-validation";
     public static final String ADD_OUTPUT_SAM_PROGRAM_RECORD = "add-output-sam-program-record";
     public static final String ADD_OUTPUT_VCF_COMMANDLINE = "add-output-vcf-command-line";
+    public static final String REANNOTATE_VCF_LONG_NAME = "reannotate-vcf";
```

Comment:
I think the declaration of this constant in `FuncotatorArgumentDefinitions` below is sufficient, and you don't need to repeat it here in `StandardArgumentDefinitions`